doc/reference/std/errors.md

```diff
@@ -426,7 +426,23 @@ displaying the exception with `display-exception`).
 ```
 
 Invokes `thunk` with an exception handler that dumps the exception
-stack trace with `dump-stack-trace!`.
+stack trace with `dump-stack-trace!`
+if `(dump-stack-trace?)` is true (the default).
+
+### dump-stack-trace?
+```scheme
+(define dump-stack-trace? (make-parameter #t))
+```
+A parameter that controls whether `with-exception-stack-trace`
+will actually dump a stack trace to standard error.
```

Comment:
that's not all of it, is it?

Reply:
Why not?

Reply:
the primordial exception handler consults it too, no?

Reply:
The one from `spawn-actor` only ... but what on the main thread?

Reply:
it should affect that too, otherwise inconsistency!

Reply:
So yeah, let's remove the wording about with-exception-stack-trace, it is the least important part; the important part is controlling printing of stack traces when displaying exceptions.

Reply:
FIXME?